elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -108,13 +107,13 @@ $pfe-cta--Color--fallback: #003366;
 
 ::slotted(*) {
   display: inline;
-  color: pfe-local(Color, $pfe-cta--Color--fallback) !important;  // because of redhat-theme styles on links
+  color: #{pfe-local(Color, $pfe-cta--Color--fallback)} !important;  // because of redhat-theme styles on links
   text-decoration: none;
   &::before {  // inner border on priority CTAs
     content: "";
     display: block;
     box-sizing: border-box;
-    border: 1px solid var(--pfe-cta__inner--BorderColor);
+    border: 1px solid var(--pfe-cta__inner--BorderColor);;
```

Comment:
There's a double colon at the end of this line. To use the mixin for this too: `#{pfe-local($cssvar: BorderColor, $region: inner)}`

Reply:
updated; thanks!